src/plugins/rovpilot/public/js/rovpilot.js

```diff
@@ -379,11 +379,7 @@
       //to deal with the clocks not being in sync between the computer and the ROV.
       var self = this;
 
-      //Initial sync of state
-      self.cockpit.rov.emit('plugin.rovpilot.getState', function(state) {
```

Comment:
Not actually sure why this code was here. Either it was left over from a prior refactor or I missed the a change where we moved the state of power level to the ROV which would not make sense.